src/ComponentPanel.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react';
+import {imagePath} from './constants';
+
+class ComponentPanel extends Component {
```

Comment:
I think we should rename this to ComponentCatalogPanel. Then ButtonCatalog makes more sense